docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -173,34 +173,24 @@ Since the file `./src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/fun
 5) Build the tester image again, deploy noobaa, and run the test (repeat the steps starting from 'Build Core And Tester Images (Noobaa-Core)' above).
 
 #### B. Temporary change - this change will be saved in the file inside the container, useful when you need a small change.
-1) Find container ID: `minikube ssh docker container ls | grep test`
-2) Enter container as the root user: `minikube ssh "docker container exec -it -u 0 <Container ID> /bin/bash"`
-3) Change file permissions: `chmod 777 ./src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py`
-
+You can edit the test by going to the test file and editing the test function. e.g. if you are working on test `s3tests_boto3.functional.test_s3:test_set_bucket_tagging` then you should `vi ./src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py` and search for the function `test_set_bucket_tagging`.
 ## Compare to AWS Response (Inside Tester Pod)
 Prerequisites:
 Following the 'Run a Single Ceph S3 Test' steps until 'Deploy The Tester Deployment (Noobaa-Core Tab)'.
 
 In this section we will do some manual changes that will allow you to check AWS response for a specific test (tests that do not use neither ACL nor tenant group).
-1) Find container ID: `minikube ssh docker container ls | grep test`
-2) Enter container as the root user: `minikube ssh "docker container exec -it -u 0 <Container ID> /bin/bash"` (We use root user because we want to change content of read-only files).
-3) Go to noobaa working directory: `cd /root/node_modules/noobaa-core/`
-4) Run the script that will create the necessary accounts in noobaa and update the Ceph S3 tests config file accordingly: `node ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_config_setup.js`
-5) Change the configuration file to match AWS details, `vi ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_config_setup.js`:
+1) copy configuration file:
+  ```bash
+  cp .src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf .src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf.aws
```

Comment:
I would change the new configuration name instead of `test_ceph_s3_config.conf.aws` as written to `test_ceph_s3_config_aws.conf` to leave the extension as `.conf` to be easier for humans to read.